po/zh-TW.po

```diff
@@ -9028,16 +9028,9 @@ msgid ""
 "  you can iterate over a vector with `for i in some_vec { .. }` but\n"
 "  `some_vec.next()` doesn't exist."
 msgstr ""
-"*`Iterator` 特徵會對集合實作許多常見的函式程式操作，\n"
-"例如 `map`、`filter`、`reduce` 等等。您可以藉由此特徵找出\n"
-"所有相關的說明文件。在 Rust 中，這些函式會產生程式碼，且應與對應的命令式\n"
-"實作項目一樣有效率。\n"
+"*`Iterator` 特徵會對集合實作許多常見的函式程式操作，例如 `map`、`filter`、`reduce` 等等。您可以藉由此特徵找出所有相關的說明文件。在 Rust 中，這些函式會產生程式碼，且應與對應的命令式實作項目一樣有效率。\n"
```

Comment:
In this case, I think the problem is not the newline, but the lack of space between "*" and the list item content. Also let's try to keep lines short for people with smaller screens.

```suggestion
"* `Iterator` 特徵會對集合實作許多常見的函式程式操作，例如 `map`、`filter`、"
"  `reduce` 等等。您可以藉由此特徵找出所有相關的說明文件。在 Rust 中，這些"
"  函式會產生程式碼，且應與對應的命令式實作項目一樣有效率。\n"
```